From a275ea7e48516a625f2cd4fe968db93383f9bdd2 Mon Sep 17 00:00:00 2001
From: Wolfgang Breyha <wbreyha@gmx.net>
Date: Sat, 19 Oct 2019 20:04:06 +0200
Subject: [PATCH] pps-tools: install timepps.h in location specified by RFC
 2783

This is need by eg. gpsd to build with proper PPS support.
It was already submitted once in #2892 which ended in a stalemate.
Signed-off-by: Wolfgang Breyha <wbreyha@gmx.net>
---
 utils/pps-tools/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/pps-tools/Makefile b/utils/pps-tools/Makefile
index 695114760cb..d5bfd897ad9 100644
--- a/utils/pps-tools/Makefile
+++ b/utils/pps-tools/Makefile
@@ -33,8 +33,8 @@ endef
 TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/timepps.h $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/usr/include/sys
+	$(CP) $(PKG_BUILD_DIR)/timepps.h $(1)/usr/include/sys/
 endef
 
 define Package/pps-tools/install
